src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -29,6 +29,12 @@ public Task Stats()
 
                 var stats = new DatabaseStatistics();
                 stats.LastDocEtag = DocumentsStorage.ReadLastDocumentEtag(context.Transaction.InnerTransaction);
+
+                if (stats.LastDocEtag != null)
+                {
+                    stats.LastChangeVector = Database.DocumentsStorage.GetDocumentsFrom(context, stats.LastDocEtag.Value).FirstOrDefault()?.ChangeVector;
```

Comment:
What we are tracking is the last change vector received through replication (found it out the hard way, before introducing the new value)